packages/api/src/schema/inputTypes.js

```diff
@@ -19,6 +19,15 @@ export default [
       key: String!
       value: SortDirection!
     }
+    input WorkQueueSortOptionInput {
+      key: String!
+      value: SortDirection!
+    }
+
+    input EventSortOptionInput {
```

Comment:
one remark:

why duplicate and not refactor it to have a single input type "SortOptionInput"? instead of ProductReviewSortOptionInput, WorkQueueSortOptionInput, EventSortOptionInput that all look the same?

Now you implement another concept with the events rather than using one of the existing concepts of orderBy in searchAssortments or sort in reviews (i'd prefer the review way).